### PR TITLE
Integrate the broker's source with gradle

### PR DESCRIPTION
**Pull request by ununhexium**
Partially closes https://github.com/sovity/PMO-Software/issues/972

```[tasklist]
### Checklist
- [ ] The PR title is short and expressive.
- [ ] I have updated the CHANGELOG.md. See [changelog_update.md](https://github.com/sovity/authority-portal/tree/main/docs/dev/changelog_updates.md) for more information.
- [ ] I have updated the Deployment Migration Notes Section in the CHANGELOG.md for any configuration / external API changes.
- [ ] I have performed a **self-review**
```

### CODE REVIEW

File: extensions/broker-server-api/client/build.gradle.kts [hunk 1]
```diff
@@ -65,14 +67,15 @@ tasks.getByName<org.openapitools.generator.gradle.plugin.tasks.GenerateTask>("op
             "annotationLibrary" to "swagger1",
             "hideGenerationTimestamp" to "true",
             "useRuntimeException" to "true",
-    ))
+        )
+    )
 
-    inputSpec.set("${project.buildDir}/${openapiFile}")
+    inputSpec.set("${targetLocation.resolve(openapiFileName)}")
```
Comment (richardtreier) [inline extensions/broker-server-api/client/build.gradle.kts:73]: can unwrap template

File: extensions/broker-server-api/client/build.gradle.kts [hunk 2]
```diff
@@ -95,14 +98,31 @@ checkstyle {
...
+    dependsOn(postprocessGeneratedClient)
+    dependsOn(JavaPlugin.JAVADOC_TASK_NAME)
+    archiveClassifier.set("javadoc")
+    from(tasks["javadoc"])
+}
+
+artifacts {
+    add("archives", sourcesJar)
+    add("archives", javadocJar)
 }
```
Comment (richardtreier) [inline extensions/broker-server-api/client/build.gradle.kts:121]: can't this be handled in the submodules clause of the root gradle file?

File: extensions/broker-server-postgres-flyway-jooq/build.gradle.kts [hunk 3]
```diff
@@ -22,37 +22,37 @@ val postgresVersion: String by project
...
+    alias(libs.plugins.jooq)
     `java-library`
     `maven-publish`
 }
 
 dependencies {
-    api("org.jooq:jooq:3.18.7")
-    api("com.github.t9t.jooq:jooq-postgresql-json:4.0.0")
+    api(libs.jooq.jooq)
+    api(libs.t9tJooq)
```
Comment (richardtreier) [inline extensions/broker-server-postgres-flyway-jooq/build.gradle.kts:38]: library should be called "jooqPostgresJsonExt"

**Review by richardtreier — CHANGES_REQUESTED** [review]
great job on making the gradle builds more type-safe

File: extensions/broker-server-api/client/build.gradle.kts (hunk 2 above)
Comment (ununhexium) [reply extensions/broker-server-api/client/build.gradle.kts:121]: It could, but if I put it in the root gradle, all the projects will depend on `postprocessGeneratedClient`, which is API-wrapper specific.

File: extensions/broker-server-postgres-flyway-jooq/build.gradle.kts (hunk 3 above)
Comment (ununhexium) [reply extensions/broker-server-postgres-flyway-jooq/build.gradle.kts:38]: Ok, but it will be under `t9tJooq` so no need to indicate `...Ext`

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,9 +7,17 @@ please see [changelog_updates.md](docs/dev/changelog_updates.md).
 
 ### Overview
 
+Starting from version `8`, the Broker has been merged with the Community edition.
```
Comment (ununhexium) [inline CHANGELOG.md:10]: Maybe this doesn't need to marked as a major change.

For the user, the only news is that the broker's versioning line changes.
The feature set remains the same.

File: launchers/Dockerfile [hunk 5]
```diff
@@ -35,3 +37,5 @@ CMD ["start"]
 
 # health status is determined by the availability of the /health endpoint
 HEALTHCHECK --interval=5s --timeout=5s --retries=10 CMD curl --fail http://localhost:11001/api/check/health
+# TODO: do I need the API key?
```
[the remark this answers is not in the document]
Comment (ununhexium) [reply launchers/Dockerfile]: Fixed

File: .github/workflows/release_docs_zip.yml [hunk 6]
```diff
@@ -17,7 +17,7 @@ jobs:
         run: |
           ARCHIVE_FILE_NAME="broker-server-release-${GITHUB_REF#refs/tags/v}-deployment-docs.zip"
           echo "ARCHIVE_FILE_NAME=$ARCHIVE_FILE_NAME" >> $GITHUB_ENV
-          zip -r -q $ARCHIVE_FILE_NAME README.md CHANGELOG.md docker-compose.yaml .env connector/.env connector/Dockerfile connector/README.md
+          zip -r -q $ARCHIVE_FILE_NAME README.md CHANGELOG.md docker-compose-broker.yaml .env launchers/.env.broker launcher/Dockerfile connector/README.md
```
Comment (richardtreier) [inline .github/workflows/release_docs_zip.yml:20]: let's merge the broker, 11000, 22000 should remain connectors, 33000 can become the broker. for this we should especially double check both the local development and local demo deployment guides and ensure that they contain all the ports, URLs, etc.

File: .gitignore [hunk 7]
```diff
@@ -39,7 +39,7 @@ build
 
 **/.env
 !.env
-!launchers/.env
+!launchers/.env.extensions
 !connector/.env
```
Comment (richardtreier) [inline .gitignore:43]: this looks like there might be an .env.broker missing. but the question is, do we actually want to ignore it? What's this line? 😄

File: CHANGELOG.md (hunk 4 above)
Comment (richardtreier) [reply CHANGELOG.md:10]: We definitely should

Because once the refactoring is done, we should do a major release where we align all the versions (connector, UI, broker) to the common next highest major version, of course with an explanation why versions were skipped.

this is for the strategic goal of aligning versions

a TODO for the next release would be then to also adjust the version VARs in .env files, release processes, (future)  changelog entries so that we can have a streamlined "one community edition version"

File: README.md [hunk 8]
```diff
@@ -79,6 +79,15 @@ We packaged critical extensions for compatibility with our EDC UI and general us
...
 
+## sovity Community Edition Broker extension
+
+[//]: # (TODO: check if this is still the same version at releasing.)
+Our EDC Broker indexes EDCs to offer a convenient centralized search engine. It is part of this project since version 8.
+
+See [here](launchers/README.md) for a list of our sovity Community Edition Broker Docker images.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```
Comment (richardtreier) [inline README.md:90]: We should remove this.

Honestly, we don't need to adjust the main description text just yet, which is what I would've suggested as an alternative, as the broker will soon be dissolved

File: SECURITY.md [hunk 9]
```diff
@@ -29,4 +29,5 @@ This information will help us triage your report more quickly.
 We prefer all communications to be in English.
 
 ## Attribution
-This file is adapted from [eclipse-edc](https://github.com/eclipse-edc/DataDashboard) project.
\ No newline at end of file
+
+This file is adapted from [eclipse-edc](https://github.com/eclipse-edc/DataDashboard) project.
```
Comment (richardtreier) [inline SECURITY.md:33]: `from the [eclipse-edc/DataDashboard](...) project`

File: archived/broker/README.md [hunk 10]
```diff
@@ -99,7 +99,7 @@ the [docker-compose.yaml](#local-demo).
 
 ### Local Demo
 
-There is a [docker-compose.yaml](docker-compose.yaml) that starts a broker and a connector.
+There is a [docker-compose-broker.yaml](../../docker-compose-broker.yaml) that starts a broker and a connector.
```
Comment (richardtreier) [inline archived/broker/README.md:102]: this file should be obsolete once the docker composes are merged and the local demo guide itself was adjusted

unfortunately we need to do that until we are doing the feature for the MDS, so we can properly sell them that we merged the repos.

File: docs/deployment-guide/goals/broker-local-demo/README.md [hunk 11]
```diff
@@ -0,0 +1,34 @@
+# Deployment Goal: Broker Local Demo
```
Comment (richardtreier) [inline docs/deployment-guide/goals/broker-local-demo/README.md:1]: Local Demos will be merged

File: extensions/broker-server-api/client/build.gradle.kts (hunk 2 above)
Comment (richardtreier) [reply extensions/broker-server-api/client/build.gradle.kts:116]: These code blocks are task adjustments, so they are additive, right?

What I meant, is, that we could enable source jars for all jars by moving this to the submodules directive:


```kt
// root build.gradle.kts submodules section
val sourcesJar by tasks.creating(Jar::class) {
    dependsOn(JavaPlugin.CLASSES_TASK_NAME)
    archiveClassifier.set("sources")
    from(sourceSets["main"].allSource)
}

val javadocJar by tasks.creating(Jar::class) {
    dependsOn(JavaPlugin.JAVADOC_TASK_NAME)
    archiveClassifier.set("javadoc")
    from(tasks["javadoc"])
}
```

maybe in the root just the default withSourcesJar() would be sufficient

```kt
// api wrapper build.gradle.kts
val sourcesJar by tasks.creating(Jar::class) {
    dependsOn(postprocessGeneratedClient)
}

val javadocJar by tasks.creating(Jar::class) {
    dependsOn(postprocessGeneratedClient)
}
```

File: extensions/broker-server-postgres-flyway-jooq/build.gradle.kts [hunk 12]
```diff
@@ -5,7 +5,7 @@ import org.testcontainers.containers.PostgreSQLContainer
 
 val jooqDbType = "org.jooq.meta.postgres.PostgresDatabase"
 val jdbcDriver = "org.postgresql.Driver"
-val postgresContainer = "postgres:11-alpine"
+val postgresContainer = "postgres:15-alpine"
```
Comment (richardtreier) [inline extensions/broker-server-postgres-flyway-jooq/build.gradle.kts:8]: in the customer portal project I moved this to the libs.versions.toml aswell, and it worked great, fully DRY
 
I don't know how much we can DRY the postgres version here, so please decide if it's worth to move this into the versions.toml

File: extensions/broker-server/src/test/java/de/sovity/edc/ext/brokerserver/db/TestDatabaseViaEnv.java [hunk 13]
```diff
@@ -17,6 +17,10 @@
 import org.apache.commons.lang3.Validate;
 import org.junit.jupiter.api.extension.ExtensionContext;
 
+/**
+ * @deprecated Use test containers wherever possible
+ */
+@Deprecated
```
Comment (richardtreier) [inline extensions/broker-server/src/test/java/de/sovity/edc/ext/brokerserver/db/TestDatabaseViaEnv.java:23]: we should either completely purge it, or leave it in (compare the TestDatabaseFactory, where you removed the env stuff.

there might still be some use cases, where you want to use a persistent local DB that you're connected to.

File: gradle.properties [hunk 14]
```diff
@@ -1,26 +1,9 @@
 sovityEdcExtensionsVersion=0.0.1-SNAPSHOT
 sovityEdcExtensionGroup=de.sovity.edc.ext
+sovityBrokerServerGroup=de.sovity.broker
 sovityEdcGroup=de.sovity.edc
 edcGroup=org.eclipse.edc
 edcVersion=0.2.1.2
```
Comment (richardtreier) [inline gradle.properties:6]: can't this also be part of the libs.versions.toml?

File: gradle.properties [hunk 15]
```diff
@@ -1,26 +1,9 @@
 sovityEdcExtensionsVersion=0.0.1-SNAPSHOT
 sovityEdcExtensionGroup=de.sovity.edc.ext
+sovityBrokerServerGroup=de.sovity.broker
 sovityEdcGroup=de.sovity.edc
 edcGroup=org.eclipse.edc
```
Comment (richardtreier) [inline gradle.properties:5]: same with the edc dependencies

File: launchers/Dockerfile [hunk 16]
```diff
@@ -10,6 +10,7 @@ USER edc:edc
 
 # Which app.jar to include
 ARG CONNECTOR_NAME="sovity-ce"
+ARG CONNECTOR_TYPE="extensions"
```
Comment (richardtreier) [inline launchers/Dockerfile:13]: `extensions -> connector`

File: launchers/README.md [hunk 17]
```diff
@@ -35,12 +35,139 @@ functionalities for self-hosting purposes.
...
+    </td>
+  </tr>
+  <tr>
+    <td>
+      <a href="https://github.com/sovity/edc-broker-server-extension/pkgs/container/broker-server-dev">broker-dev</a>
+    </td>
+    <td>Development</td>
+    <td>
+      <ul>
+        <li>Local Deployment via our `docker-compose.yaml`</li>
```
Comment (richardtreier) [inline launchers/README.md:142]: backticks don't work. Let's also call it "Local demos" as we do above for the connector

File: launchers/connectors/broker-server-dev/build.gradle.kts [hunk 18]
```diff
@@ -0,0 +1,38 @@
...
+    implementation("${edcGroup}:data-plane-selector-core:${edcVersion}")
+    implementation("${edcGroup}:api-observability:${edcVersion}")
+    implementation("${edcGroup}:configuration-filesystem:${edcVersion}")
+    implementation("${edcGroup}:control-plane-aggregate-services:${edcVersion}")
+    implementation("${edcGroup}:http:${edcVersion}")
+    implementation("${edcGroup}:dsp:${edcVersion}")
+    implementation("${edcGroup}:json-ld:${edcVersion}")
+
+    // JDK Logger
+    implementation("${edcGroup}:monitor-jdk-logger:${edcVersion}")
```
Comment (richardtreier) [inline launchers/connectors/broker-server-dev/build.gradle.kts:22]: do we re-use the logging style of the connector?

Did you check, with which system / style the broker CE did logging before and compared it to the connector CE?

For example, our EE does JSON logging which would definitely differ

**Review by richardtreier — CHANGES_REQUESTED** [review]

File: extensions/broker-server-api/client/build.gradle.kts (hunk 2 above)
Comment (richardtreier) [reply extensions/broker-server-api/client/build.gradle.kts:116]: i meant in the bottom block where you "get" tasks of the given class / type / name to adjust them once more

Comment (ununhexium) [reply extensions/broker-server-api/client/build.gradle.kts]: ok, got it. But there's an shorter way, see
https://github.com/sovity/edc-extensions/pull/921/files#diff-c0dfa6bc7a8685217f70a860145fbdf416d449eaff052fa28352c5cec1a98c06R125

Comment (ununhexium) [reply extensions/broker-server-api/client/build.gradle.kts]: And now we have sources and javadocs for all the deliveries. ^_^

File: CHANGELOG.md (hunk 4 above)
Comment (ununhexium) [reply CHANGELOG.md:10]: Noted for next release
https://github.com/sovity/edc-extensions/issues/928#issuecomment-2117525942

File: .github/workflows/ci.yml [hunk 19]
```diff
@@ -65,6 +65,7 @@ jobs:
           image-base-name: ${{ env.IMAGE_BASE_NAME }}
           image-name: "edc-dev"
           connector-name: "sovity-dev"
+          connector-type: "extensions"
```
[the remark this answers is not in the document]
Comment (ununhexium) [reply .github/workflows/ci.yml]: Fixed

File: .editorconfig [hunk 20]
```diff
@@ -8,3 +8,18 @@ end_of_line = lf
...
+[*.md]
+max_line_length = off
+trim_trailing_whitespace = false
+indent_size = 2
+
+[*.{yml,yaml}]
+indent_size = 2
+
+[docker-compose.yaml]
+indent_size = 2
```
[the remark this answers is not in the document]
Comment (ununhexium) [reply .editorconfig:25]: yes

File: .dockerignore [hunk 21]
```diff
@@ -38,7 +38,8 @@ build
 *.hprof
 
 .env
-!launchers/.env
+!launchers/.env.extensions
```
[the remark this answers is not in the document]
Comment (ununhexium) [reply .dockerignore]: yes

File: .github/actions/build-connector-image/action.yml [hunk 22]
```diff
@@ -19,6 +19,9 @@ inputs:
   connector-name:
     required: true
     description: "EDC Connector Name in launchers/connectors/{connector-name}"
+  connector-type:
+    required: true
+    description: "Type of connector: 'extensions' or 'broker'"
```
[the remark this answers is not in the document]
Comment (ununhexium) [reply .github/actions/build-connector-image/action.yml]: yes

File: .github/actions/build-connector-image/action.yml [hunk 23]
```diff
@@ -19,6 +19,9 @@ inputs:
   connector-name:
     required: true
     description: "EDC Connector Name in launchers/connectors/{connector-name}"
+  connector-type:
```
[the remark this answers is not in the document]
Comment (ununhexium) [reply .github/actions/build-connector-image/action.yml:22]: Changed

File: .github/workflows/ci.yml [hunk 24]
```diff
@@ -98,8 +101,33 @@ jobs:
...
+        with:
+          registry-url: ${{ env.REGISTRY_URL }}
+          registry-user: ${{ env.REGISTRY_USER }}
+          registry-password: ${{ secrets.GITHUB_TOKEN }}
+          image-base-name: ${{ env.IMAGE_BASE_NAME }}
+          image-name: "broker-server-dev"
+          connector-name: "broker-server-dev"
+          connector-type: "broker"
+          title: "Broker Server (Dev)"
+          description: "EDC IDS Broker Server. This dev version contains no persistence or auth and can be used to quickly start a locally running Broker Server + Broker UI."
```
[the remark this answers is not in the document]
Comment (ununhexium) [reply .github/workflows/ci.yml]: fixed

File: .github/workflows/ci.yml [hunk 25]
```diff
@@ -98,8 +101,33 @@ jobs:
...
+        with:
+          registry-url: ${{ env.REGISTRY_URL }}
+          registry-user: ${{ env.REGISTRY_USER }}
+          registry-password: ${{ secrets.GITHUB_TOKEN }}
+          image-base-name: ${{ env.IMAGE_BASE_NAME }}
+          image-name: "broker-server-ce"
+          connector-name: "broker-server-ce"
+          connector-type: "broker"
+          title: "Broker Server (Community Edition)"
+          description: "EDC IDS Broker Server. Contains DB extensions and requires dataspace credentials to join an existing dataspace."
```
[the remark this answers is not in the document]
Comment (ununhexium) [reply .github/workflows/ci.yml]: ok

File: .github/workflows/license_scan.yml [hunk 26]
```diff
@@ -31,6 +31,8 @@ jobs:
         run: cd extensions/wrapper/clients/typescript-client && npm install
       - name: npm install (typescript-client-example)
         run: cd extensions/wrapper/clients/typescript-client-example && npm install
+      - name: npm install (client-ts)
+        run: cd extensions/broker-server-api/client-ts && npm install
```
[the remark this answers is not in the document]
Comment (ununhexium) [reply .github/workflows/license_scan.yml:35]: Same for the 2 npm commands above?

[the remark this answers is not in the document]
Comment (ununhexium) [reply .github/workflows/license_scan.yml:35]: Fixed

File: .gitignore (hunk 7 above)
Comment (ununhexium) [reply .gitignore]: Yes, the broker is the line below with the previous naming: connector/.env

And we *don't* want to ignore it.

File: README.md (hunk 8 above)
Comment (ununhexium) [reply README.md:90]: okay, removed.

File: extensions/broker-server-postgres-flyway-jooq/build.gradle.kts (hunk 12 above)
Comment (ununhexium) [reply extensions/broker-server-postgres-flyway-jooq/build.gradle.kts:8]: DRYed it. It needed a bit of code generation but now there's only 1 instance of that image name.

File: extensions/broker-server/src/test/java/de/sovity/edc/ext/brokerserver/db/TestDatabaseViaEnv.java (hunk 13 above)
Comment (ununhexium) [reply extensions/broker-server/src/test/java/de/sovity/edc/ext/brokerserver/db/TestDatabaseViaEnv.java:23]: ok, deprecation removed

File: gradle.properties (hunk 15 above)
Comment (ununhexium) [reply gradle.properties:5]: ok, cleared

I also found a lot of deps that I didn't switch to the new format

File: launchers/Dockerfile (hunk 16 above)
Comment (ununhexium) [reply launchers/Dockerfile]: yes

File: launchers/README.md (hunk 17 above)
Comment (ununhexium) [reply launchers/README.md:142]: fixed

File: launchers/connectors/broker-server-dev/build.gradle.kts (hunk 18 above)
Comment (ununhexium) [reply launchers/connectors/broker-server-dev/build.gradle.kts]: The `logging.properties` of the broker and the launchers are identical.

Comment (ununhexium) [reply launchers/connectors/broker-server-dev/build.gradle.kts]: And they both used `monitor-jdk-logger`.

The broker dev and server used the same config file:
https://github.com/sovity/edc-broker-server-extension/blob/main/connector/build.gradle.kts

So both with the JDK/monitor lib.

So unless there's some funky config at runtime I think they will log the same way.

File: .github/workflows/ci.yml [hunk 27]
```diff
@@ -22,6 +22,22 @@ jobs:
     permissions:
       contents: read
       packages: write
+
+    services:
```
[the remark this answers is not in the document]
Comment (ununhexium) [reply .github/workflows/ci.yml]: done

File: .github/workflows/release_docs_zip.yml (hunk 6 above)
Comment (ununhexium) [reply .github/workflows/release_docs_zip.yml]: Done

File: .gitignore (hunk 7 above)
Comment (ununhexium) [reply .gitignore]: And it should have been launchers/.env.broker

File: SECURITY.md (hunk 9 above)
Comment (ununhexium) [reply SECURITY.md:33]: fixed

File: docs/deployment-guide/goals/broker-local-demo/README.md (hunk 11 above)
Comment (ununhexium) [reply docs/deployment-guide/goals/broker-local-demo/README.md:1]: Merged

File: gradle.properties (hunk 14 above)
Comment (ununhexium) [reply gradle.properties]: sure

File: archived/broker/README.md (hunk 10 above)
Comment (ununhexium) [reply archived/broker/README.md]: Merged

File: .github/workflows/release_docs_zip.yml [hunk 28]
```diff
@@ -17,7 +17,7 @@ jobs:
         run: |
           ARCHIVE_FILE_NAME="broker-server-release-${GITHUB_REF#refs/tags/v}-deployment-docs.zip"
           echo "ARCHIVE_FILE_NAME=$ARCHIVE_FILE_NAME" >> $GITHUB_ENV
-          zip -r -q $ARCHIVE_FILE_NAME README.md CHANGELOG.md docker-compose.yaml .env connector/.env connector/Dockerfile connector/README.md
+          zip -r -q $ARCHIVE_FILE_NAME README.md CHANGELOG.md docker-compose.yaml .env launchers/.env.broker launcher/Dockerfile connector/README.md
```
Comment (richardtreier) [inline .github/workflows/release_docs_zip.yml:20]: I believe we should remove the deployment zip. We had this for releasing the broker when the repo was closed source. I think before we have the ZIPs for the broker floating around in our CE releases, we should remove this step.

File: docs/deployment-guide/goals/broker-local-demo/README.md (hunk 11 above)
Comment (richardtreier) [inline docs/deployment-guide/goals/broker-local-demo/README.md:1]: remove file

File: docs/deployment-guide/goals/production/4.2.0/README.md [hunk 29]
```diff
@@ -1,4 +1,4 @@
-Deployment Goal: Production
+eployment Goal: Production
```
Comment (richardtreier) [inline docs/deployment-guide/goals/production/4.2.0/README.md:1]: typo

File: extensions/broker-server/README.md [hunk 30]
```diff
@@ -16,7 +16,7 @@
 
 ## About this Extension
 
-Implementation of an IDS Broker Server as an EDC Extension.
+Implementation of an Broker Server as an EDC Extension.
```
Comment (richardtreier) [inline extensions/broker-server/README.md:19]: grammar: "of a Broker"

will also suggest rewording: "EDC Broker backend as an EDC Extension."

File: utils/versions/build.gradle.kts [hunk 31]
```diff
@@ -0,0 +1,56 @@
+import com.squareup.javapoet.FieldSpec
+import com.squareup.javapoet.JavaFile
+import com.squareup.javapoet.TypeName
+import com.squareup.javapoet.TypeSpec
+import javax.lang.model.element.Modifier.*
```
Comment (richardtreier) [inline utils/versions/build.gradle.kts:5]: no asterisk imports, this might help: https://github.com/sovity/customer-portal/blob/6cf9736bfd602dd80fb403377d2a95fe5e8f63ce/backend/.editorconfig#L15-L20

**Review by richardtreier — CHANGES_REQUESTED** [review]

File: docs/deployment-guide/goals/broker-local-demo/README.md (hunk 11 above)
Comment (ununhexium) [reply docs/deployment-guide/goals/broker-local-demo/README.md:1]: yes

File: extensions/broker-server/README.md (hunk 30 above)
Comment (ununhexium) [reply extensions/broker-server/README.md:19]: fixed

File: utils/versions/build.gradle.kts (hunk 31 above)
Comment (ununhexium) [reply utils/versions/build.gradle.kts]: yes that helped.

**Review by tmberthold — COMMENTED** [review]
Please also pay attention to the effects of the runtime of GH-actions, we just had this topic: The CI currently seems to run for 20-35 minutes for this PR.

https://github.com/sovity/edc-extensions/actions/workflows/ci.yml?query=branch%3A972-group-edc-ce-ui-broker-part3.1

**Review by richardtreier — APPROVED** [review]
LGTM
